app/helpers/katello/katello_url_helper.rb

```diff
@@ -18,6 +18,7 @@ def file_prefix?(url)
     private
 
     def valid_for_prefixes(url, prefixes)
+      return false unless URI.parse(url).host.present?
       return false unless (scheme = URI.parse(url).scheme).present?
```

Comment:
@johnpmitsch 

I'm not sure I agree with moving these to the same line. They are checking different parts of the URI. Also there is a nested assignment. This gets messy and hard to parse when reading back over the code, imo.